### ISSUE TITLE
marker-layer clickable doesn't seem to work

### ISSUE THREAD

**Issue by Techn1x** [miguelcobain/ember-leaflet#155]
Looking at the documentation here, it seems as though there is a 'clickable' option that can be binded to
http://www.ember-leaflet.com/docs/marker-layer

However it seems like it doesn't work? When I do this, my marker is still clickable, and shows the popup
`{{#marker-layer location=asset.location clickable="false"}}`
or
`{{#marker-layer location=asset.location clickable=false}}`

I did some googling and it looks like in the 1.0-beta1 of leaflet.js, the 'clickable' option changed to 'interactive'
https://github.com/Leaflet/Leaflet/blob/master/CHANGELOG.md
`Renamed clickable option to interactive (by @AndriiHeonia). #2838 #2499`

I gave interactive="false" a go as well and that didn't work either.

Is this an issue in my implementation or a bug of ember-leaflet?

**Comment by billdami** [comment]
@Techn1x it looks like ember-leaflet has not yet been updated to account for this change to the leaflet.js API. A quick fix would be to extend the ember-leaflet `marker-layer` component in your app (and any other components you need the `interactive` option for), and override the `leafletOptions` property, removing `clickable`, and adding `interactive`. e.g.

```js
import MarkerLayer from 'ember-leaflet/components/marker-layer';

export default MarkerLayer.extend({
  leafletOptions: Object.freeze([
    'icon', 'interactive', 'draggable', 'keyboard', 'title',
    'alt', 'zIndexOffset', 'opacity', 'riseOnHover', 'riseOffset'
  ])
});
```
https://github.com/miguelcobain/ember-leaflet/blob/master/addon/components/marker-layer.js#L13

**Comment by miguelcobain** [comment]
Released ember-leaflet `3.0.16` which includes the `interactive` property on marker-layer.

Thank you for reporting!

**Comment by billdami** [comment]
@miguelcobain fyi, i think the same change also applies to the other layer types that have the `clickable` option. (i.e. `path-layer`, `geojson-layer`)

**Comment by Techn1x** [comment]
Also, perhaps clickable should be removed as an option on `marker-layer` (and other layers) so as not to confuse

**Comment by miguelcobain** [comment]
@billdami i published `3.0.17` that includes interactive-layer which all path-layers, image and marker layers extend. It also includes the new video-layer from newer leaflet. Thanks for pointing that out.

@Techn1x ember-leaflet tries to work with past version of leaflet. Removing clickable could break for anyone using ember-leaflet with leaflet 0.7 or even some version of 1.0.
